File: repo_php/PHP_POO/heritage_example_4/example_one/Pessoa.php
<?php
    class Pessoa {
        protected $nome;
        protected $idade;
        protected $sexo;

        function __construct($nome){
            $this -> setNome($nome);
        }

        public function fazerAniverssario (){
            $this-> idade ++;
        }

        public function getNome(){
            return $this->nome;
        }

        public function setNome($nome){
            $this->nome = $nome;
        }

        public function getIdade(){
            return $this->idade;
        }

        public function setIdade($idade){
            $this->idade = $idade;
        }

        public function getSexo(){
            return $this->sexo;
        }

        public function setSexo($sexo){
            $this->sexo = $sexo;
        }
}

File: java/java-oo/java-io/java-io/src/main/java/teste/java/io/Pessoa.java
package teste.java.io;

import java.io.Serial;
import java.io.Serializable;

public class Pessoa implements Serializable {
    @Serial
    private static final long serialVersionUID = 4871139191131378300L;
    String nome;
    Integer idade;

    public Pessoa(String nome, Integer idade){
        this.nome = nome;
        this.idade = idade;
    }

    public String getNome() {
        return nome;
    }

    public void setNome(String nome) {
        this.nome = nome;
    }

    public Integer getIdade() {
        return idade;
    }

    public void setIdade(Integer idade) {
        this.idade = idade;
    }
}
